### ISSUE TITLE
Move theory vignette to Literature navbar on website

### ISSUE THREAD

**Issue by jamesmbaazam** [epiverse-trace/bpmodels#137]
We currently have two "modelling" categories under the Articles tab on the package website (see screenshot). I suggest moving the recently added [theory vignette](https://epiverse-trace.github.io/bpmodels/articles/theoretical_background.html) under the "Literature" navbar. This will require collapsing the following https://github.com/epiverse-trace/bpmodels/blob/42a8831a690bcafea8762976bd898269041628e7/_pkgdown.yml#L39-L46. 
<img width="685" alt="Screenshot 2023-09-01 at 11 34 59" src="https://github.com/epiverse-trace/bpmodels/assets/43169674/7c26a3f5-07aa-45ff-8faf-ee28da026aaf">

**Comment by sbfnk** [comment]
Good idea - alternatively perhaps we could have a "Background" section for theory/literature and another section called "Examples" or "Case studies" for worked through code?

**Comment by jamesmbaazam** [comment]
Would the latter be different from the vignette(s)? Also to point out ongoing work by the RFs in the [case studies](https://github.com/epiverse-trace/casestudies) repo and some potential duplicated work.

**Comment by jamesmbaazam** [comment]
I have currently implemented the proposed solution (see attached). I noticed the vignette titles were too long, so I shortened them.
<img width="488" alt="Screenshot 2023-09-01 at 12 02 30" src="https://github.com/epiverse-trace/bpmodels/assets/43169674/2e092daf-8e01-43b9-a3b0-5631b68ae214">

**Comment by sbfnk** [comment]
Looks good to me

**Comment by jamesmbaazam** [comment]
Awesome. PR incoming.
